app/org/sagebionetworks/bridge/models/accounts/SignIn.java

```diff
@@ -24,6 +25,9 @@ private SignIn(String studyId, String email, Phone phone, String password, Strin
         this.reauthToken = reauthToken;
     }
     
+    // Serializing this property as study allows us to use SignIn to construct the JSON payload that we 
+    // are accepting from clients, which is useful for tests to avoid manually constructing the JSON string.
+    @JsonProperty("study")
```

Comment:
Our tests assume that SignIn will only ever be deserialized, never serialized. This annotation implies this is no longer true. Should we add serialization tests in SignInTest?

Reply:
Added serialization test to SignInTest.